### ISSUE TITLE
[Bug.Minor] Creeper/Anticreeper terrain spawn count overlapping (can be seen on Creeper Hollow 4 map)

### ISSUE THREAD

**Issue by HenryFBP** [KnuckleCracker/CW4-bug-tracker#486]
1. The vast majority of issues encountered will go away after a reboot of your computer. This is our experience over many games over many years. Please try that. If the issue persist across a restart of your  computer, then proceed. 

2.  **It is very important that you NOT restart the game before you save the output log**
 Log file location: 
   ** Windows 10** (currently the only supported platform)
      The log file location is here under Windows 10 (currently the only supported platform). 
           `c:\Users\YOUR_USER\AppData\LocalLow\Knuckle Cracker\Creeper World 4\output_log.txt`
      Alternately, copy and paste this short-form file location into the address bar of Windows File Explorer
             `%LOCALAPPDATA%Low\Knuckle Cracker\Creeper World 4`

[output_log.txt](https://github.com/KnuckleCracker/CW4-bug-tracker/files/5706599/output_log.txt)

3. Screenshots help! 

![image](https://user-images.githubusercontent.com/3236324/102435422-32a59700-3fdc-11eb-9eaf-5308551caf67.png)

4. Describe the bug 

Basically, Creeper/Anticreeper terrain spawn counts are overlapping. You can see it in the lower left. Can be seen on Creeper Hollow 4 map.

Running on Windows 10, 1080p and full screen. Default monitor scale.

**Comment by knucracker** [comment]
Not sure I follow. That thing circled represents the duty cycle for the creeper terrain. How long it is on, vs off.

**Comment by HenryFBP** [comment]
I think I might be misinterpreting the UI control in that case.

I thought that there were 2 elements overlapping, but if that's supposed to just show 1 number, then you can close this issue as it's not an issue. Thank you!

**Comment by knucracker** [comment]
Yeah, the number shows the rate of emission. The bars shows the duty cycle; how long it is on relative to being off. It's subtle and information overkill and almost never useful. Not sure why I did it :)
